src/aideck/loaders/text.py

```
@@ -0,0 +1,116 @@
+            ".c", ".cpp", ".h", ".cs", ".go", ".rs", ".php", ".rb",
+            ".css", ".ini"
+        ]
+
+    def load(self, source: Union[str, Path, List[Union[str, Path]]]) -> List[Document]:
+        """
+        Loads all text documents from the given source synchronously.
+
+        This is a convenience wrapper around the async `aload` method.
+        """
+        try:
+            loop = asyncio.get_running_loop()
```

Ruff (0.13.1)

Local variable `loop` is assigned to but never used

Remove assignment to unused variable `loop`

(F841)